app/controllers/admin/schools_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Admin
+  class SchoolsController < Admin::BaseController
+    skip_after_action :verify_authorized, only: :index
```

Comment:
Do we need to skip it? Does `policy_scope` not authorize?

Reply:
Nope, there are two, verify_authorized and verify_policy_scoped